data/schema.yaml

```diff
@@ -61,72 +61,159 @@ components:
     article:
       type: object
       required:
-        - lastUpdatedDateTime
-        - locator
-        - metaDescription
-        - model
-        - passport
-        - publishedDateTime
-        - seoHeadline
-        - type
+        - content
+        - metadata
+        - promo
       properties:
-        lastUpdatedDateTime:
-          example: 2017-12-21 17:32:28 +00:00
-          format: date-time
-          type: string
-        locator:
-          example: "urn:bjk:hello:asset:c000zk2l88wo"
-          format: uuid
-          type: string
-        metaDescription:
-          example: "story about a young brave hero"
-          type: string
-        model:
+        content:
+          type: object
+          required:
+            - model
           properties:
-            blocks:
-              $ref: "#/components/schemas/blocks"
+            model:
+              type: object
+              required:
+                - blocks
+              properties:
+                blocks:
+                  $ref: "#/components/schemas/blocks"
+        metadata:
           required:
-            - blocks
-          type: object
-        passport:
+            - blockId
+            # - blockTypes
+            - created
+            - firstPublished
+            - id
+            - lastPublished
+            - lastUpdated
+            - locators
+            # - metaDescription
+            - passport
+            - tags
+            - type
           properties:
-            articleType:
-              enum:
-                - analysis
-                - feature
-                - news
-              example: analysis
+            blockId:
               type: string
-            home:
-              example: "http://www.bbc.co.uk/ontologies/passport/home/News"
+            blockTypes:
+              type: object # why isn't this an array when the json is loaded? Does the json need to be parsed?
```

Comment:
This should be an array but in the data is an object, I believe this is because we aren't parsing the JSON but assigning it as a JS object. To be handled in a future issue